fs/dafsstream/dafsstream.cpp

```diff
@@ -855,14 +888,25 @@ class CDFUPartReader : public CDaFileSrvClientBase, implements IDFUFilePartReade
             requestNode->setPropBool("outputGrouped", preserveGrouping);
         }
 
+        // JCSMORE these are defaults, but should be picked up from file->queryFileDescriptor()
```

Comment:
Is there a Jira to do so? Is this comment still valid?

Reply:
Hope opened a JIRA for this: https://track.hpccsystems.com/browse/HPCC-22322